Dockerfile

```diff
@@ -98,7 +98,7 @@ RUN mkdir ${ANDROID_HOME} && chown --recursive ${USER} ${HOME_DIR} ${ANDROID_HOM
 USER ${USER}
 
 # Download and install android's NDK/SDK
-COPY ci/makefiles/android.mk /tmp/android.mk
+COPY --chown=user:user ci/makefiles/android.mk /tmp/android.mk
 RUN make --file /tmp/android.mk \
     && sudo rm /tmp/android.mk
```

Comment:
Not entirely sure why we `chown`-ed, but I guess that `sudo` here is no longer needed then

Reply:
I ran into a `Permission Denied` error during the docker build. It originated from the make command. I don't like running to much commands in `sudo` so I tried to fix the rights first. After the change it worked.

I ran the Dockerbuild without `sudo` entirely and had no issues. So perhaps the adding of sudo right is entirely unnecessary?

Reply:
We should definitely not run the docker client with sudo indeed.
What I meant in my comment is regarding the `sudo` line within the container. Could we remove it since you chown user:user. So basically two lines after your change there's a `sudo rm` simply remove the sudo part

Reply:
Yes I could even run dockerbuild & build a test apk with the following changes:
```
...
# with sudo access and no password 
# RUN usermod -append --groups sudo ${USER}
# RUN echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers

...

# Download and install android's NDK/SDK
COPY --chown=user:user ci/makefiles/android.mk /tmp/android.mk
RUN make --file /tmp/android.mk \
    && rm /tmp/android.mk
...
```
I'm not sure about further consequences but for my limited test the user didn't even need to be in the `sudoers` group.